lib/plugins/aws/info/getStackInfo.js

```diff
@@ -57,6 +44,24 @@ module.exports = {
       }
 
       return BbPromise.resolve();
+    })
+    .then(() => this.provider.getAccountId())
+    .then((accountId) => {
+      const stage = this.provider.getStage();
+      const region = this.provider.getRegion();
+
+      this.gatheredData.info.functions = [];
+
+      this.serverless.service.getAllFunctions().forEach((func) => {
+        const functionInfo = {};
+        const name = `${this.serverless.service.service}-${stage}-${func}`;
```

Comment:
Yep that works too. But maybe that would make the service class AWS specific rather than provider agnostic? I'm sensing that we need a method in the provider class, but not worth it considering the upcoming changes 😉